.evergreen/config.yml

```diff
@@ -642,6 +642,12 @@ functions:
         working_dir: "src"
         script: |
           ${PREPARE_SHELL}
+          if [ "${skip_EC2_auth_test}" = "true" ]; then
+             echo "This platform does not support the EC2 auth test, skipping..."
+             exit 0
+          fi
+          # Remove any prepare_mongodb_aws.sh files created by other tasks .
+          rm "${PROJECT_DIRECTORY}/prepare_mongodb_aws.sh"
```

Comment:
I found this oversight when working on the OIDC testing.  We weren't actually testing EC2 credential fetch.

Reply:
Updated